### PR TITLE
tools, media: add user guide for sync-diff-inspector

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
preview at
https://github.com/WangXiangUSTC/docs-cn/blob/xiang/sync-diff-inspector/tools/sync-diff-inspector.md

### CODE REVIEW

File: tools/sync-diff-inspector.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
+---
+title: sync-diff-inspector 用户文档
+category: tools
+---
+
```
Comment (lilin90) [inline tools/sync-diff-inspector.md]: We should add a first-level heading and put the "title" here. Keep the title consistent with that in metadata.

File: tools/sync-diff-inspector.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+## sync-diff-inspector 简介
+sync-diff-inspector 是一个用于校验 MySQL／TiDB 中两份数据是否一致的工具，该工具提供了修复数据的功能（适用于修复少量不一致的数据）。
+
+主要功能：
+* 对比表结构和数据
```
Comment (lilin90) [inline tools/sync-diff-inspector.md]: This list displays well on GitHub, but does not work on PingCAP website. Please add a blank line before this line.

File: tools/sync-diff-inspector.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+* 如果数据不一致，则生成用于修复数据的 sql
+* 支持多个表的数据与单个表数据的比较（针对分库分表同步数据到总表的场景）
+* 支持不同库名／表名的数据的比较
+
+Github 地址：[sync-diff-inspector](https://github.com/pingcap/tidb-tools/tree/master/sync_diff_inspector)
```
Comment (lilin90) [inline tools/sync-diff-inspector.md]: Github -> GitHub

File: tools/sync-diff-inspector.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+title: sync-diff-inspector 用户文档
+category: tools
+---
+
+## sync-diff-inspector 简介
+sync-diff-inspector 是一个用于校验 MySQL／TiDB 中两份数据是否一致的工具，该工具提供了修复数据的功能（适用于修复少量不一致的数据）。
+
+主要功能：
+* 对比表结构和数据
+* 如果数据不一致，则生成用于修复数据的 sql
```
Comment (lilin90) [inline tools/sync-diff-inspector.md]: sql -> SQL

File: tools/sync-diff-inspector.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+* 支持不同库名／表名的数据的比较
+
+Github 地址：[sync-diff-inspector](https://github.com/pingcap/tidb-tools/tree/master/sync_diff_inspector)
+
+下载地址：[sync-diff-inspector-linux-amd64.tar.gz](https://download.pingcap.org/sync-diff-inspector-linux-amd64.tar.gz)
+
+## sync-diff-inspector 的使用
+### 通用配置文件说明
+
+```
```
Comment (lilin90) [inline tools/sync-diff-inspector.md]: Perhaps add `toml` here for this code block?

File: tools/sync-diff-inspector.md [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+
+### 分库分表场景下数据对比的配置示例：
+假设有两个 MySQL 实例，使用同步工具同步到一个 TiDB 中，场景如图所示：
+
+![shard-table-sync](../media/shard-table-sync.png)
+
+如果需要检查同步后数据是否一致，可以使用如下的配置对比数据：
+
+```
```
Comment (lilin90) [inline tools/sync-diff-inspector.md]: Add `toml`?

File: tools/sync-diff-inspector.md [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+host = "127.0.0.3"
+port = 4000
+user = "root"
+password = ""
+```
+
+### 运行 sync-diff-inspector
+执行如下命令：
+
+```
```
Comment (lilin90) [inline tools/sync-diff-inspector.md]: Add `bash`?

**Comment by WangXiangUSTC** [comment]
Thanks @lilin90

**Pull request merged**